packages/admin-ui/src/filters/filter-search.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+import { BaseFilter } from './filter-base'
+import { ComboboxItem } from '../combobox/combobox-item'
+import { FilterRadio } from './filter-radio'
+import { FilterSeachbox } from './filter-searchbox'
+import { SingleItemLabel } from './SingleItemLabel'
+import { Box } from '..'
+import { useMessageFormatter } from '../i18n'
+import type { FilterProps } from '.'
+
+import { messages } from './filter.i18n'
+import * as style from './filter.style'
+
+export function FilterSearch<T>(props: FilterProps<T>) {
+  const formatMessage = useMessageFormatter(messages.searchBox)
+  const {
+    state: { combobox, appliedItem, baseId, getOptionId, getOptionLabel },
+    state,
+  } = props
+
+  const currentSelectedId = combobox.selectedItem
+    ? getOptionId(combobox.selectedItem)
+    : null
```

Comment:
Is `null` the better option here? Probably an empty string would be better? What do you think?